public/css/globals/_variables.styl

```diff
@@ -5,7 +5,7 @@
 /* Typefaces */
 
 $font-facit = 'jaf-facitweb', 'Helvetica Neue', Helvetica, Arial, sans-serif
-$font-inconsolata = 'inconsolata', menlo, consolas, monospaced
+$font-inconsolata = 'inconsolata', menlo, consolas, monospaced, monospace
```

Comment:
I was not sure, that's why I added instead of replacing it. It just updated my PR ✌